optimize/util/optimize-commons/pom.xml

```diff
@@ -77,25 +77,31 @@
       <artifactId>json-path</artifactId>
       <version>${jsonpath.version}</version>
     </dependency>
+    <!-- unused/declared
     <dependency>
       <groupId>org.yaml</groupId>
       <artifactId>snakeyaml</artifactId>
       <version>2.3</version>
     </dependency>
+    -->
 
     <!-- ClassGraph  -->
+    <!-- unused/declared
     <dependency>
       <groupId>io.github.classgraph</groupId>
       <artifactId>classgraph</artifactId>
       <version>4.8.175</version>
     </dependency>
+    -->
 
     <!-- Logging -->
+    <!-- unused/declared
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
       <version>${slf4j.version}</version>
     </dependency>
```

Comment:
I would like us to understand whether this can actually be removed. It might be to allow a flexible log adaptor